package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "dust-loader",
-	"version": "1.0.1",
+	"version": "1.0.2",
```

Comment:
This is a feature addition, not a bugfix, so please update the version to `1.1.0`